conbench/db.py

```diff
@@ -20,10 +21,16 @@
 log.info("psycopg2.__libpq_version__: %s", psycopg2.__libpq_version__)
 
 
+# Pick less log verbosity when
+logfunc = log.info
+if "pytest" in sys.modules:
+    logfunc = log.debug
```

Comment:
haha, isn't this what thresholds are for? So we could set the log level to DEBUG in pytest and INFO otherwise.

But I don't mind this, since it's so limited in scope.